intellst-api/config/security.yaml

```diff
@@ -0,0 +1,27 @@
+security:
```

Comment:
this file isn't needed as it duplicates the config in config/packages/security.yml